users/models.py

```diff
@@ -50,13 +50,23 @@ def create_superuser(self, email, first_name,
                                 **other_fields)
 
 
+class UserType(models.TextChoices):
+    OWNER = 'O', 'Owner'
+    SEEKER = 'S', 'Seeker'
+
+
 class User(AbstractBaseUser, PermissionsMixin):
 
     email = models.EmailField(unique=True)
     first_name = models.CharField(max_length=50)
     last_name = models.CharField(max_length=50)
     date_joined = models.DateTimeField(default=timezone.now)
     birth_date = models.DateField(auto_now=False, auto_now_add=False)
+    user_type = models.CharField(
```

Comment:
Does every Owner have an Apartment record, and every Seeker have a Seeker record?
If so, this field doesn't give you anything and can be misleading. (for example, if you have an apratment record, but this user was changed to a seeker.

Reply:
every Owner has an Apartment record, and every Seeker has a Seeker record.
But I still don't see the problem with this addition.
It is only meant for the programmer to be able to identify whether the user is an Owner or a Seeker, and handle him fittingly. (A user does not have an apartment field, an apartment has a user field)
For example, if a user logged in I would like to know whether he is an owner or seeker, and show him a fitting home page.
If I try to use `user.apartment`, and the user is actually a seeker with no apartment record, I will get an error. So every time I will have to use try-except, catch the error, and show him a seeker page. Despite that I can just use the field to know he is a seeker to begin with.

Reply:
@simzacks And regarding your example, if you have an apartment record, but this user was changed to a seeker, the apartment record will get deleted, a seeker record will be created and attached to the user. (There will be maintenance, We wont let a situation like that happen.)

Reply:
Generally you want to leave that kind of information out of the database model because it is duplicated data.
In other words, I know who is an apartment user and I know who is a Seeker. I don't need a field with data.

Reply:
@simzacks I now understand that this causes data duplication. 
But the thing is that we weren't able to overcome the obstacle of finding out whether a user is an Owner or Seeker.
(in a comfortable and easy way)
Will a method that checks this be OK? Or is there another way to solve this and you suggest we just leave this as is?
I will close this PR.

Reply:
I'm not sure why you want to know what kind of user every user is.
If you want to know who all the apartment users are then get
Apartments.objects.all() and from there you go into the user records.
if you want all the seekers, you get Seekers.objects.all() and from there you get the user record.

When will you ever want a list of all the users in the system whith a flag as to what kind of user?

Reply:
Its not a list that I will desire @simzacks .
It's more for a situation as such:
A user logged in, he needs to be directed to the hompage. BUT - Owners and Seekers have different home pages. So, im order to understand which html I should present to him, I could use the user_type field for that.
It looked like a better idea than looking through the entire Seekers.objects.all() and Owner.objects.all() in order to find this user. But it seems as I maybe getting something here wrong, right?

Reply:
My mistake. OneToOne relationships can be accessed from both directions.

https://docs.djangoproject.com/en/3.1/topics/db/examples/one_to_one/

In this example, you can see that Place can access restaurant and Restaurant can access place.

So here you can get the user and if he has an apartment send him one way and if he was a seeker send him the other way.

Reply:
This opened up my eyes
Thank you!!